### ISSUE TITLE
Nested mutation to tables with 1:1 relationships allows creation with multiple nested rows.

### ISSUE THREAD

**Issue by Munksgaard** [mlipscombe/postgraphile-plugin-nested-mutations#7]
I have the following two tables:

```
CREATE TABLE post.post (
  id SERIAL PRIMARY KEY,
  text TEXT NOT NULL
);

CREATE TABLE post.post_image (
  id SERIAL PRIMARY KEY,
  post_id INTEGER UNIQUE NOT NULL REFERENCES post.post(id) ON DELETE CASCADE,
  url TEXT NOT NULL
);
```

The plugin allows the user to add a list of images to a post instead of just one image, even though it's clearly not allowed by the database. 

Here's the resulting graphql schema:
[post.txt](https://github.com/mlipscombe/postgraphile-plugin-nested-mutations/files/2615773/post.txt)

**Comment by mlipscombe** [comment]
This needs introspection changes to see if there is a relevant unique constraint, and then if so, not creating the `create` input type as a `GraphQLList`.  I'll put it on the TODO list.

**Comment by mlipscombe** [comment]
This should be fixed in v1.0.0.alpha13.  Feel free to reopen if anything's not right.
